src/components/scene/background.js

```diff
@@ -5,7 +5,8 @@ var COMPONENTS = require('../../core/component').components;
 module.exports.Component = register('background', {
   schema: {
     color: {type: 'color', default: 'black'},
-    transparent: {default: false}
+    transparent: {default: false},
+    generateEnvironment: {default: true}
```

Comment:
Do we want the default to be true?

Reply:
I think it's pretty safe and not a huge performance hit as it only renders the once. It will even fix materials unexpectedly not looking right because they expect an environment map. So it's a nice feature, that you only need to type 'background` to get.

I'd even be tempted to take it a step further and make `background="transparent:true"` a default component on `<a-scene>` or at least add `background` to the hello-world boilerplate just to get the better behaviour by default.